packages/csharp/PortingAssistant/PortingAssistant.Common/Services/AssesmentService.cs

```diff
@@ -42,7 +42,7 @@ public class AssessmentService : IAssessmentService
 
                 if (solutionAnalysisResult.IsCompletedSuccessfully)
                 {
-                    TelemetryCollectionUtils.CollectSolutionMetrics(solutionAnalysisResult, request, startTime, tgtFramework);
+                    TelemetryCollectionUtils.CollectSolutionMetrics(solutionAnalysisResult.Result, request, startTime, tgtFramework);
```

Comment:
My understanding is that `solutionAnalysisResult` getting passed in is a reference. Changing it to `SolutionDetails` is just changing to a different reference value, and it doesn't save any extra memories. Also since `SolutionDetails` is a property of `solutionAnalysisResult`, the garbage collection would work the same regardless which reference value we pass in.